src/main/java/de/plushnikov/intellij/plugin/activity/LombokProjectValidatorActivity.java

```diff
@@ -79,29 +81,47 @@ public void runActivity(@NotNull Project project) {
     if (hasLombokLibrary && !annotationProcessorsEnabled &&
       ProjectSettings.isEnabled(project, ProjectSettings.IS_ANNOTATION_PROCESSING_CHECK_ENABLED, true)) {
 
-      String annotationProcessorsConfigName = new AnnotationProcessorsConfigurable(project).getDisplayName();
+      suggestEnableAnnotations(project, group);
+    }
+  }
 
-      Notification notification = group.createNotification(LombokBundle.message("config.warn.annotation-processing.disabled.title"),
-        LombokBundle.message("config.warn.annotation-processing.disabled.message", project.getName()),
-        NotificationType.ERROR,
-        new SettingsOpeningListener(project, annotationProcessorsConfigName));
+  private void suggestEnableAnnotations(Project project, NotificationGroup group) {
+    String annotationProcessorsConfigName = new AnnotationProcessorsConfigurable(project).getDisplayName();
```

Comment:
This variable looks like being obsolete, it is not used anywhere?

Reply:
Yeah I've forgotten about it. I'll remove it :)

Reply:
fixed